sklearn/compose/_target.py

```diff
@@ -41,7 +41,9 @@ class TransformedTargetRegressor(RegressorMixin, BaseEstimator):
 
         transformer.inverse_transform(regressor.predict(X))
 
-    Read more in the :ref:`User Guide <transformed_target_regressor>`.
+    For an example use case of the TransformedTargetRegressor, refer to
```

Comment:
```suggestion
    For an example use case of the :class:`TransformedTargetRegressor`, refer to
```

Reply:
Hm, this is the docstring of :class:`TransformedTargetRegressor`, so the link would link to the same document that the user currently looks at. I believe that this is unnecessary, isn't it?

Reply:
Alternatively: 

```suggestion
    For an example use case refer to
```

Reply:
this can also go bellow the Examples section, but no strong opinions here